virtcontainers/README.md

```diff
@@ -64,27 +64,27 @@ testing purposes.
 ## virtcontainers and Kubernetes CRI
 
 `virtcontainers`'s API is loosely inspired by the Kubernetes [CRI][cri] because
-we believe it provides the right level of abstractions for containerized pods.
+we believe it provides the right level of abstractions for containerized sandboxs.
 However, despite the API similarities between the two projects, the goal of
 `virtcontainers` is _not_ to build a CRI implementation, but instead to provide a
 generic, runtime-specification agnostic, hardware-virtualized containers
 library that other projects could leverage to implement CRI themselves.
 
 # Design
 
-## Pods
+## Sandboxs
```

Comment:
Needs to handle `Pods` to `Sandboxes` but not `Sandboxs`
I though kata had spell detection, but looks like it hasn't.

Reply:
Oh yeah good point here. @bergwolf You need to change `sandboxs` into `sandboxes`

Reply:
Yeah, I'll fix them.